yarn-project/end-to-end/src/e2e_p2p/slashing.test.ts

```diff
@@ -24,14 +24,17 @@ const DATA_DIR = fs.mkdtempSync(path.join(os.tmpdir(), 'slashing-'));
 
 // This test is showcasing that slashing can happen, abusing that our nodes are honest but stupid
 // making them slash themselves.
+// TODO: this test looks to slash attesters because ANY epoch was pruned.
+// But soon (TODO(#14407), TODO(#14408)) we will only slash epoch prune due to data withholding OR
+// the data was available and the epoch could have been proven.
 describe('e2e_p2p_slashing', () => {
```

Comment:
Cool to see test so much smaller. Good with the todos. I would say please run the slashing amount where we crank it up to a LOT, to make sure that the slashing client actually survives big numbers 😬.